packages/api/core/src/api/init-scripts/init-npm.ts

```diff
@@ -7,16 +7,21 @@ import username from 'username';
 import { setInitialForgeConfig } from '../../util/forge-config';
 import installDepList from '../../util/install-dependencies';
 import { readRawPackageJson } from '../../util/read-package-json';
+import { version } from '../../../package.json';
```

Comment:
> (@electron-forge/core) $ node_modules/.bin/tsc --emitDeclarationOnly src/api/init-scripts/init-npm.ts(10,25): error TS2307: Cannot find module '../../../package.json'.

We should use `fs` to read the file and then parse it instead of using `import`